plugins/Jobs.cpp

```diff
@@ -1290,3 +1319,41 @@ bool BedrockPlugin_Jobs::_isValidSQLiteDateModifier(const string& modifier) {
     // Matched all parts, valid syntax
     return true;
 }
+
+void BedrockPlugin_Jobs::handleFailedReply(const BedrockCommand& command) {
+    if (SIEquals(command.request.methodLine, "GetJob") || SIEquals(command.request.methodLine, "GetJobs")) {
+
+        list<string> jobIDs;
+        if (SIEquals(command.request.methodLine, "GetJob")) {
+            STable jobJSON = SParseJSONObject(command.response.content);
+            if (jobJSON.find("jobID") != jobJSON.end()) {
+                jobIDs.push_back(jobJSON["jobID"]);
+            }
+        } else {
+            STable jobsJSON = SParseJSONObject(command.response.content);
+            list<string> jobs = SParseJSONArray(jobsJSON["jobs"]);
+            for (auto& job : jobs) {
+                STable jobJSON = SParseJSONObject(job);
+                if (jobJSON.find("jobID") != jobJSON.end()) {
+                    jobIDs.push_back(jobJSON["jobID"]);
+                }
+            }
+        }
+        SWARN("Failed sending response to '" << command.request.methodLine << "', re-queueing jobs: "<< SComposeList(jobIDs));
+        if(_server) {
```

Comment:
Can this be false? If so, what would happen with the jobs? Maybe we should alert in the else saying we could not revert the changes?